internal/downloader/downloader.go

```diff
@@ -21,9 +21,17 @@ package downloader
 import (
 	"context"
 	"fmt"
+	"regexp"
+	"strings"
 
 	"github.com/open-policy-agent/conftest/downloader"
 	log "github.com/sirupsen/logrus"
+
+	e "github.com/hacbs-contract/ec-cli/pkg/error"
+)
+
+var (
+	DL001 = e.NewError("DL001", "Attempting to download from unsecure source", e.ErrorExitStatus)
```

Comment:
I was wondering about unsecure vs insecure. I think maybe insecure is correct, but there are people arguing either way in https://english.stackexchange.com/questions/19653/insecure-or-unsecure-when-dealing-with-security so :shrug: .

```suggestion
	DL001 = e.NewError("DL001", "Attempting to download from insecure source", e.ErrorExitStatus)
```